### PR TITLE
CE-3566 Style the InfoboxBuilder entry point

### PR DESCRIPTION
**Pull request by adamkarminski**
@RafLeszczynski @lukaszkonieczny

### CODE REVIEW

File: extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxInsertDialog.js [hunk 1]
```diff
@@ -245,22 +245,24 @@ ve.ui.WikiaInfoboxInsertDialog.prototype.createEmptyStateContent = function ( un
...
+	this.$body.html( $content ).append( this.addNewTemplateWidget() );
 };
 
 ve.ui.WikiaInfoboxInsertDialog.prototype.addNewTemplateWidget = function () {
-    var button = new OO.ui.ButtonWidget( {
-        label: 'Add new template',
-        icon: 'add',
-        iconTitle: 'Add new template'
-    } );
+    var container = $( '<div>', { class: 've-ui-wikiaInfoboxInsertDialog-add-container' } ),
```
Comment (Grunny) [inline extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxInsertDialog.js]: Conventions: jQuery vars should be prefixed with `$`. Also, spaces -> tabs.

File: resources/lib/oojs-ui/oojs-ui.js [hunk 2]
```diff
@@ -2728,6 +2728,9 @@ OO.ui.WindowManager.static.sizes = {
 	},
 	'800px': {
 		width: 800
+	},
+	page: {
```
Comment (adamkarminski) [inline resources/lib/oojs-ui/oojs-ui.js:2732]: I'm working on moving it to our WindowManager

Comment (adamkarminski) [inline resources/lib/oojs-ui/oojs-ui.js:2732]: Or is it not worth it? I'd either modify the VE core or the OO UI one... @Grunny ?

Comment (Grunny) [inline resources/lib/oojs-ui/oojs-ui.js:2732]: I'd consider modifying VE to be preferable. The OOjs UI library really shouldn't have anything specific to Oasis in it, and any changes to its functionality that aren't generic should go in VE modules that extend it.

**Pull request merged**